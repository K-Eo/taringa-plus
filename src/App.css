.blue-bg {
  background-color: #004a95;
}

.app-decorator {
  height: 13px;
}

.app-decorator-top {
  border-radius: 10px 10px 0 0;
}

.app-decorator-bottom {
  border-radius: 0 0 10px 10px;
}

.app-wrapper {
  padding-left: 12px;
  padding-right: 12px;
}

.app-container {
  background: #fff;
  padding: 10px;
  border-radius: 0 0 5px 5px;
}

@keyframes placeholderShimmer {
  0% {
    background-position: -200px 0
  }

  100% {
    background-position: 200px 0
  }
}

.bg-masker {
  background: white;
  position: absolute;
}

.panel-default {
  border: none;
}

.panel {
  border-radius: 5px;
  margin-bottom: 15px;
}

.panel:last-child {
  margin-bottom: 0;
}

.panel-default .panel-heading {
  background: rgba(217,217,217,1);
  background: linear-gradient(to bottom, rgba(217,217,217,1) 0%, rgba(191,191,191,1) 50%, rgba(179,179,179,1) 100%);

  color: #464646;
  font-size: 12px;
  font-weight: bold;
  line-height: 1.3em;
  padding: 5px 10px;
  text-shadow: 0 1px 0 #ccc;
}

.panel-default .panel-body {
  background: #e7e7e7;
  padding: 8px;
  border-radius: 0 0 5px 5px;
  line-height: 1.3em;
}

.app-container .row {
  margin-right: -5px;
  margin-left: -5px;
}

.app-container .row > div[class^='col-'] {
  padding-right: 5px;
  padding-left: 5px;
}
