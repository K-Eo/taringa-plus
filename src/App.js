import React, { Component } from 'react';
import { connect } from 'react-redux'
import { Container } from 'semantic-ui-react'
import './App.css'
import Nav from './Nav'
import Section from './sections/components/Section'

import { sectionsSelector } from './posts/selectors'
import * as actions from './posts/actions'

class App extends Component {
  render() {
    const { sections, fetch } = this.props

    return (
      <div>
        <Nav/>
        <Container className='app-container'>
          {
            sections.map(section => (
              <Section
                key={section.id}
                {...section}
                fetch={fetch}
              />
            ))
          }
        </Container>
      </div>
    );
  }
}

const mapStateToProps = (state) => ({
  sections: sectionsSelector(state)
})

export default connect(mapStateToProps, actions)(App);
