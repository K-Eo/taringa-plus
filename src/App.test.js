import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

it('Renders App', () => {
  expect(true).toBe(true)
})
