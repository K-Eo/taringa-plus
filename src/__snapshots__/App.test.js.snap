// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App 1`] = `
<div>
  <div
    className="ui large inverted top fixed menu"
  >
    <div
      className="ui container"
    >
      <div
        className="header item"
        onClick={[Function]}
      >
        Taringa Plus!
      </div>
    </div>
  </div>
</div>
`;
